glommio/src/executor/placement/mod.rs

```diff
@@ -163,6 +163,18 @@ pub enum Placement {
     Custom(Vec<CpuSet>),
 }
 
+impl Clone for Placement {
```

Comment:
The tests are great, but why do we need to implement this manually?
I figured just deriving Clone would work

Reply:
I just spaced it and had "implement the trait" stuck in my head.

Reply:
Np. Let's use derive though because it is cleaner, but happy to take the tests. I had sent a PR for this as well, will close mine in favor of yours